### ISSUE TITLE
Unable to login w/ 2FA enabled

### ISSUE THREAD

**Issue by dylan-apex** [tryretool/retool-cli#19]
When I run `retool login` and select  "Google SSO" as the login method and after going through the Google authentication, I get this error in the command line: `HTTP Request Error: { success: true, message: 'FIDO2 Not Setup' }`

After I reset the 2FA in the admin panel and ran `retool login` again it worked without any issues.

Other notes
- I've only signed up/logged in with Google SSO, so I have no username/password to login with
- My 2FA method was a FIDO2 Security Key and nothing else
- I did not try cookie authentication

**Comment by PeteTheHeat** [comment]
Some clarifications:

- You had 2FA enabled, tried to login via Google SSO, and hit the `'FIDO2 Not Setup'` error message.
- After resetting 2FA, logging in via Google SSO worked, and you were unblocked.

This all correct?

I'm not sure why you hit this, haven't had other reports of this yet.

**Comment by dylan-apex** [comment]
Yeah that is correct. To clarify, by "reset" 2FA, I mean that I removed it entirely.

After going through Google OAuth signin, it opens up `http://localhost:3020/auth` with the access token and xsrf token as params, but the page is already unreachable. Going to the command line it says:
```
dylan@DESKTOP:~$ retool login
? How would you like to login? Log in using Google SSO in a web browser


HTTP Request Error: { success: true, message: 'FIDO2 Not Setup' }

Failed to make request to https://<domain>.retool.com/api/user.
```

I re-added my FIDO2 key and was able to recreate this after doing `retool logout` and `retool login`

**Comment by aldegoeij** [comment]
I have a similar error, have a user with no SSO or Google magical, just username + password + 2FA OTP

getting:
```bash
⠧ Verifying Retool DB credentials

HTTP Request Error: { success: true, message: '2FA Challenge' }

Failed to make request to https://space.org.retool.com/api/resources.
```

**Comment by connorguy** [comment]
I have the same issue as above. I also tried cookie login and got an error with that too:

```
retool login
? How would you like to login? Log in by pasting in cookies
? What is your Retool origin? (e.g., https://my-org.retool.com). https://<url>.retool.com
? What is your XSRF token? (e.g., 26725f72-8129-47f7-835a-cba0e5dbcfe6)
  Log into Retool, open cookies inspector.
  In Chrome, hit ⌘+⌥+I (Mac) or Ctrl+Shift+I (Windows, Linux) to open dev tools.
  Application tab > your-org.retool.com in Cookies menu > double click cookie value and copy it. <token>
? What is your access token? It's also found in the cookies inspector. 
Error: Access token is invalid.
```
